app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details.scala.html

```diff
@@ -0,0 +1,72 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{BTIApplication, Case, CaseStatus, Decision, Event, StoredAttachment}
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.{attachments, complete_case_button}
+@import uk.gov.hmrc.tariffclassificationfrontend.utils.Dates
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{EventType, CaseStatusChange}
+
+@(c: Case, events: Seq[Event])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+    <div id="activity_heading">
+        <h3 class="heading-medium mt-0">@messages("case.menu.activity")</h3>
+    </div>
+
+    <table class="govuk-table">
+        <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+                <th class="govuk-table__header" scope="col">Name</th>
+                <th class="govuk-table__header" scope="col">Description</th>
+                <th class="govuk-table__header" scope="col">Date</th>
+
+            </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+        @for(event <- events) {
+            <tr class="govuk-table__row">
+                <td class="activity-cell">@event.operator.name</td>
+
+                @if(event.details.`type` == EventType.NOTE) {
+                <td class="activity-cell-desc">
+                    <details role="group">
+                        <summary role="button">
+                            <span>
+                                Case note added
+                            </span>
+                        </summary>
+                        <div id="case-note-comment">
+                            @event.details.comment
+                        </div>
+                    </details>
+                } else {
+                    <td>Status changed to @event.details.asInstanceOf[CaseStatusChange].to.toString.toLowerCase</td>
+                }
+
+                <td class="activity-cell">@Dates.format(event.timestamp)</td>
+            </tr>
+        }
+            <tr><td></td><td class="activity-cell">Application Submitted</td><td class="activity-cell">@Dates.format(c.createdDate)</td></tr>
+        </tbody>
+
+    </table>
+
+    <p>
+        <h3 class="heading-small activity-inline-label">@messages("case.activity.assigned-to"):</h3>
+        <span id="assigned-to">@c.assigneeId</span>
```

Comment:
What if its not assigned to anyone?

Reply:
At the moment it just shows "Assigned To: " (blank)

Reply:
Go spot. Joe has given me some text to display in the case that we have no assignee

Reply:
Done